core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -74,6 +79,21 @@ static __thread struct argv_environ_values *argv_environ;
 static __thread struct addr_pool *addr_pool;
 #endif
 
+uint64_t kFrequency = 0;
```

Comment:
could you please use `sneak_case` for variables and function names to be consistent with the rest of the project?

Reply:
What is an example of a sneak_case variable

Reply:
In this specific example it'd be `k_frequency` or just `frequency`. You can read more about it here: https://en.wikipedia.org/wiki/Snake_case

What you've used though is called `camelCase`: https://en.wikipedia.org/wiki/Camel_case